arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/continuations/result.kt

```diff
@@ -0,0 +1,51 @@
+package arrow.core.continuations
+
+import arrow.core.Either
+import arrow.core.Validated
+import arrow.core.identity
+
+public object ResultEffectScope {
```

Comment:
We cannot expose this as a `public object`, see issue: https://github.com/arrow-kt/arrow/issues/2547

Reply:
removed it 👍🏾